domain/src/Access/DomainAccessCheck.php

```diff
@@ -52,11 +52,11 @@ public function applies(Route $route) {
    * {@inheritdoc}
    */
   public function checkPath($path) {
-    $allowed_paths = $this->configFactory->get('domain.settings')->get('login_paths');
+    $allowed_paths = $this->configFactory->get('domain.settings')->get('login_paths', '/user/login\r\n/user/password');
```

Comment:
> The !empty should be added, because there is a case where people will disable this entirely and force login from the primary domain.

If I read the code correctly, the only way to disable this entirely is to clear the `login_paths` textarea in the settings form. As such my suggestion to provide the default `/user/login\r\n/user/password` fallback when `login_paths` is the wrong approach and only the `!empty` check is needed.